docs/repository-management.md

```diff
@@ -0,0 +1,134 @@
+# Repository Management
+
+The goal is for this to be a living document explaining how we collaboratively manage the Gutenberg repository. If you’d like to suggest a change, please open an issue for discussion or submit a pull request to the document.
+
+This document covers:
+
+- [Issues](#issues)
+  - [Labels](#labels)
+  - [Milestones](#milestones)
+  - [Triaging Issues](#triaging-issues)
+- [Pull Requests](#pull-requests)
+  - [Code Review](#code-review)
+  - [Design Review](#design-review)
+  - [Merging Pull Requests](#merging-pull-requests)
+  - [Closing Pull Requests](#closing-pull-requests)
+
+## Issues
+
+A healthy issue backlog is one where issues are relevant and actionable. *Relevant* in the sense that they relate to the project’s current priorities. *Actionable* in the sense that it’s clear what action(s) need to be taken to resolve the issue.
+
+Any issues that are irrelevant or not actionable should be closed, because they get in the way of making progress on the project. Imagine the issue backlog as a desk: the more clutter you have on it, the more difficult it is to use the space to get work done.
+
+**Labels**
+
+To better organize the issue backlog, all issues should have [one or more labels](https://github.com/WordPress/gutenberg/labels). Here are some you might commonly see:
```

Comment:
I'd need to double-check, but I'm unsure if labels can be assigned if not being a member of the project.